content/manager_webui/default-widgets-ref.md

```diff
@@ -92,12 +97,12 @@ Displays a chart or charts (up to 5) presenting metric data for the current depl
 
 #### Configuration
 
-You must supply at least one metric or database query in the widget configuration. You also need to supply the deployment's ID, either in the page context, or by specifying it in the widget configuration. 
+You must supply at least one metric or database query in the widget configuration. You also need to supply the Deployment ID and Node instance ID, either in the page context, or by specifying it in the widget configuration. 
```

Comment:
You also need to supply > You also must enter
, either in the page context, or by specifying it in the widget configuration. > either in the page context or in the widget configuration.

Reply:
Updated.